homework/index.js

```diff
@@ -30,17 +35,122 @@
     return elem;
   }
 
+  function addRow(tBody, label, value) {
+    const row = createAndAppend('tr', tBody);
+    if (value === null) {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, {
+        text: `No ${label.toLowerCase()} is provided for this repository.`,
+      });
+    } else {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, { text: value });
+    }
+    return row;
+  }
+
+  function jsUcfirst(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
+  function createDetails(leftColumn, repository) {
+    const table = createAndAppend('table', leftColumn, { id: 'main-table' });
+    const tBody = createAndAppend('tbody', table);
+    const firstRow = addRow(tBody, 'Repository', '');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repository.html_url,
+      target: '_blank',
+      text: jsUcfirst(repository.name),
```

Comment:
While it looks user-friendly to capitalize the repository name, with this app supposedly targetting developers who use GitHub, it is important to display the information exactly as it is returned by GitHub. It can become confusing if there is a discrepancy between the case of the displayed name and the actual GitHub repo name. 

Having said that, 👍 for creating a separate function to capitalize the word (although I would call it `capitalize`).